### DISCUSSION TITLE
'reverse-alias' triple question

### DISCUSSION THREAD

**Discussion by ledgeri** [simple-login/app#871] [Q&A] [answered]
Hi!

1.) If i accidently made the mistake of sending a reverse alias adress, as it was auto-included in the reply message, can someone else use it for e.g. spam _in my name_, from a thirdparty (not actually my SL-account linked) email adress, or there is a built in block of some kind what "reverse spamfilters" the aliases, so only the real sl-account holder can send messages to the reverse alias?

2.) If i change the _**Include sender address in reverse-alias**_ setting from empty to checked, what kind of a format will be the "more readable" version? (can i get an example?)

3.) If i change the above mentioned _**Include sender address in reverse-alias**_, then i change it back, will i get new reverse-alias pack, or am i reciving the old ones....? ....what if i check the checkbox the second time>> old or new _readable_ ones?

**Comment by nguyenkims** [comment, marked as answer]
@ledgeri 

For 1) we have some measures in place to avoid a reverse alias from being used by an unauthorised user.

2) the reverse alias will include the contact address. The best way is to enable this option and try out yourself :)

3) A reverse alias is only created once so changing this option doesn't affect old reverse aliases.

**Reply by ledgeri** [reply]
Thank you for the info!
